androidApp/src/main/java/io/spherelabs/anypass/android/MainActivity.kt

```diff
@@ -10,8 +10,18 @@ import androidx.fragment.app.FragmentActivity
 import io.spherelabs.anypass.MainView
 
 class MainActivity : FragmentActivity() {
+
+    private lateinit var appUpdateManager: AppUpdateManager
```

Comment:
@alibardide5124 What do you think about to creating a other class to manage "in-app updates"? I think it's essential to follow to the Single Responsibility Principle (SRP).